demo/talkingdata-adtracking-fraud-detection/predict_server.py

```diff
@@ -162,10 +164,9 @@ def make_app():
                         default=True)
     args = parser.parse_args()
 
-    global global_args
     global_args = vars(args)
     print(global_args)
-    logging.info('init args: %s', global_args)
+    logging.info("init args: %s", global_args)
```

Comment:
Why use '' in train_and_serve.py, and use "" here? This file can use '' too.

Reply:
As the comment of `pylint`, the quote delimiters should used consistently throughout a module, may the rest of  `train_and_serve ` has more `''`, but the file of `predict_server.py` has more `'`, I think just keep consistent in whole file is ok.

Reply:
Only ~200 lines, small file, use `'` is ok

Reply:
done.